database/migrations/2018_02_09_150930_add_quantity_index_to_signups_table.php

```diff
@@ -26,7 +26,7 @@ public function up()
     public function down()
     {
         Schema::table('signups', function (Blueprint $table) {
-            $table->dropUnique('quantity');
+            $table->dropIndex('quantity');
```

Comment:
To drop an index. I believe you either have to do `$table->dropIndex(['quantity']);` (by column name) or `$table->dropIndex('signups_quantity_index');` (by index name). I would make sure the rollback works along with the migration. 

https://laravel.com/docs/5.5/migrations#dropping-indexes

Reply:
ah yes thank you @sbsmith86 !